### DISCUSSION TITLE
Series colors in Chart

### DISCUSSION THREAD

**Discussion by johannpistorius** [Live-Charts/LiveCharts2#464] [Q&A] [answered]
I'm using LiveChartsCore.SkiaSharpView.WPF Version: 2.0.0-beta.101

When adding a new series to a chart, a predetermined color is assigned to that series if a paint is not specified. Is it possible to get that color for future use?

I have the following:

~~~cs
ObservableCollection<ISeries> ChartResultsCartesian = new ObservableCollection<ISeries>();

ISeries serie = new LineSeries<double>
{
    //set properties here
};

ChartResultsCartesian.Add(serie);
~~~

which produces the following chart:

![line-series](https://user-images.githubusercontent.com/15726308/168102563-80233c0a-c6e9-4536-9b3c-f51a5607fc67.PNG)

When I change my series to a column series like so:

~~~cs
ObservableCollection<ISeries> ChartResultsCartesian = new ObservableCollection<ISeries>();

ISeries serie = new ColumnSeries<double>
{
    //set properties here
};

ChartResultsCartesian.Add(serie);
~~~

I get the following:

![column-series](https://user-images.githubusercontent.com/15726308/168102561-0769c6a3-2e29-4549-9ec0-a087ef5ff13d.PNG)

I would like to keep the same colors for both series.

Thanks

**Comment by beto-rodriguez** [comment, marked as answer]
You can just set your own:

https://lvcharts.com/docs/WPF/2.0.0-beta.220/Overview.Paints

**Reply by johannpistorius** [reply]
Thank you for taking the time to answer!
